src/js/components/Anchor/__tests__/__snapshots__/Anchor-test.tsx.snap

```diff
@@ -1444,26 +1319,24 @@ exports[`Anchor renders tag 1`] = `
   line-height: inherit;
   color: #7D4CDB;
   font-weight: 600;
-  -webkit-text-decoration: none;
   text-decoration: none;
   cursor: pointer;
 }
 
 .c1:hover {
-  -webkit-text-decoration: underline;
   text-decoration: underline;
 }
 
 <div>
   <div
     class="c0"
   >
-    <span
+    <a
```

Comment:
It looks like the `as='span'` prop in this test is being ignored

Reply:
good catch

Reply:
Fixed, needed to destructure as

Reply:
Can we make a note in our styled components google doc that for components that support `as` we need to make sure they are destructured

Reply:
Done.